prov/gni/include/gnix_freelist.h

```diff
@@ -77,6 +77,7 @@ struct gnix_freelist {
  * @param max_refill_size   Max refill size
  * @param fl                gnix_freelist
  * @return                  FI_SUCCESS on success, -FI_ENOMEM on failure
+ * @note - If the refill_size is zero the freelist is not be growable.
```

Comment:
'If the refill_size is zero, then the freelist is not growable.' 

Same thing applies to the other note.